tests/shunit/twlight_i18n_lint_test.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+shopt -s expand_aliases
+
+testBadPyNewlines() {
+    start=1
+    end=6
+    for ((i=$start; i<=$end; i++))
+    do
+        file=${TWLIGHT_HOME}/tests/shunit/data/bad_i18n_newline_$i.py
+        assertFalse "${file} should cause an error." "perl ${TWLIGHT_HOME}/bin/twlight_i18n_lint.pl ${file}"
+    done
+}
+
+testGoodPyComments() {
+    start=1
+    end=2
```

Comment:
Same here

Reply:
I rewrote this to use a pattern instead of a fixed length loop